packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -101,14 +104,19 @@ export function PanelChrome({
   const containerStyles: CSSProperties = { width, height };
   const ariaLabel = title ? selectors.components.Panels.Panel.containerByTitle(title) : 'Panel';
 
+  // Shift the hover menu down if it's on the top row so it doesn't get clipped by topnav
+  const yOffset = (gridPos?.y ?? 0) === 0 ? -16 : undefined;
+
   return (
     <div className={styles.container} style={containerStyles} aria-label={ariaLabel}>
+      {!hasHeader && menu && <HoverWidget menu={menu} title={title} offset={yOffset} />}
```

Comment:
`HoverWidget` should show up directly if `hoverHeader === true`, no other conditions. 

However, if there's nothing to show (no `dragClass`, no `title`, no `description`, no `titleItems`, no `menu`, no `streaming icon if loadingState===Streaming`) then nothing should show at all.. so it seems (because it's best to avoid such a long and specific condition if each header-related item is present) that the HoverWidget might be best implemented as wrapper around some children. The children can then be set up here in the PanelChrome. It's just a thought, not certain this is best.